events/models.py

```diff
@@ -268,9 +268,18 @@ def is_user_edited(self):
 
     def can_be_edited_by(self, user):
         """Check if current image can be edited by the given user"""
-        if user.is_superuser or user.is_regular_user_of(self.publisher):
-            return True
-        return user.is_admin_of(self.publisher)
+        if user.is_anonymous:
+            return False
+        if user.is_external and (
```

Comment:
I think it probably would make sense to also check `ENABLE_EXTERNAL_USER_EVENTS` setting here or somewhere to be on the safe side. If external users can't create events, they shouldn't be able to touch the pictures either.

Reply:
Added check here to check whether ENABLE_EXTERNAL_USER_EVENTS is enabled.